/destination_x: 10
/destination_y: 0
/velocity: 0.5